packages/katello/katello/hostname-change.rb

```diff
@@ -220,12 +220,13 @@ def using_custom_certs?(scenario_answers)
       scenario_answers["certs"]["server_key"]
     end
 
-    def delete_puppet_certs
+    def regenerate_puppet_certs
       return unless @scenario_answers['puppet'].is_a?(Hash)
 
       puppet_ssldir = @scenario_answers['puppet']['ssldir']
```

Comment:
Can we please add a return if this is `nil` to prevent `rm -rf /*/$hostname` style things? If `@old_hostname` is also empty it becomes `rm -rf /*/.*`.

Reply:
technically scope creep, but I thought the same.

updated

Reply:
or maybe not even scope creep. the rm call is different now…